src/main/java/hudson/plugins/resultscache/CachedResult.java

```diff
@@ -0,0 +1,22 @@
+package hudson.plugins.resultscache;
+
+import hudson.model.Result;
+
+import java.io.Serializable;
+
+public class CachedResult implements Serializable {
+    private static final long serialVersionUID = 1L;
+    private Result result;
+    private Integer buildNumber;
+
+    public CachedResult(Result result, Integer buildNumber) {
+        this.result = result;
+        this.buildNumber = buildNumber;
+    }
+
+    public Result getCachedResult() { return result; }
+    public Number getBuildNumber() { return buildNumber; }
```

Comment:
Do not forget to change the type in the getter.

Reply:
Oops, don't know how I missed that one. Thanks.